WalletWasabi/Helpers/IoHelpers.cs

```diff
@@ -174,5 +174,10 @@ public static void CopyFilesRecursively(DirectoryInfo source, DirectoryInfo targ
 				file.CopyTo(Path.Combine(target.FullName, file.Name));
 			}
 		}
+
+		public static void CreateEmptyFile(string path)
+		{
+			using var _ = File.Create(path);
```

Comment:
How does this behave when the file already exists? Does it throw an exception or does it over-write the file?

Reply:
https://docs.microsoft.com/en-us/dotnet/api/system.io.file.create?view=netcore-3.1#System_IO_File_Create_System_String_ it overwrites the file. But it's not clear.

Reply:
Maybe the name can hint what it does better: `CreateOrOverwriteFile(string path)`.

Reply:
Done. Good point.

Reply:
Or check if Exist and throw.